test_methods.go

```diff
@@ -33,18 +33,20 @@ const (
 )
 
 // GetMockServer is used in all the http tests.
+//
+//nolint:lll
```

Comment:
do you want to directly remove `lll` in all tests file from `.golangci.yml`?

Reply:
This isn't a test file, but yes, I do want to add this file as part of the test-file regex in `.golangci.yml`. However, I want to make a new package `starrtest` to put this in first. That requires touching every `_test.go` file in this library, so I was going to do it in another PR (after you get all yours merged). I started to do it in this one, until I realized how many things need to be changed. For now I'm okay leaving this in here; I'll re-work it later.